websub-ballerina/module.md

```diff
@@ -0,0 +1,71 @@
+## Package Overview
+
+This package contains an implementation of the W3C [**WebSub**](https://www.w3.org/TR/websub/) recommendation, which facilitates a push-based content delivery/notification mechanism between publishers and subscribers.
+
+This implementation supports introducing WebSub Subscriber, A party interested in receiving update notifications for particular topics.
+
+### Basic flow with WebSub
+1. The subscriber discovers from the publisher, the topic it needs to subscribe to and the hub(s) that deliver notifications on updates of the topic.
+
+2. The subscriber sends a subscription request to one or more discovered hub(s) specifying the discovered topic along 
+ with other subscription parameters such as:
+    - The callback URL to which content is expected to be delivered.
+    - (Optional) The lease period (in seconds) the subscriber wants the subscription to stay active.
+    - (Optional) A secret to use for [authenticated content distribution](https://www.w3.org/TR/websub/#signing-content).
+  
+3. The hub sends an intent verification request to the specified callback URL. If the response indicates 
+verification
+ (by echoing a challenge specified in the request) by the subscriber, the subscription is added for the topic at the 
+ hub.
+   
+4. The publisher notifies the hub of updates to the topic and the content to deliver is identified.
+
+5. The hub delivers the identified content to the subscribers of the topic.
+
+### Features
+
+#### Subscriber
```

Comment:
I am not sure if we really need above two topics. Because this module only talks about subscriber.

Reply:
Better to add topics like,

Subscribing 
Discovery 
Dynamic URI generation 
Running behind something like ngrok
Interoperability